internal/query/session.go

```diff
@@ -257,6 +258,14 @@ func NewSessionCreatorSearchQuery(creator string) (SearchQuery, error) {
 	return NewTextQuery(SessionColumnCreator, creator, TextEquals)
 }
 
+func NewUserIDSearchQuery(id string) (SearchQuery, error) {
+	return NewTextQuery(HumanUserIDCol, id, TextEquals)
```

Comment:
Shouldn't this be with "SessionColumnUserID" not "HumanUserIDCol"?

Reply:
You're absolutely right, I've glanced over it and noticed only `HumanUserIDCol` in the join statement